.github/workflows/master.yaml

```diff
@@ -0,0 +1,24 @@
+name: sonarcube action pr
+on:
+  push:
+    branches:
+      - master
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+        - uses: actions/checkout@v2
+
+        - name: Auth Docker GCR
+          uses: tradeshift/actions-docker-gcr/auth@master
+          with:
+            gcr-service-account-key: ${{ secrets.GCLOUD_SERVICE_ACCOUNT_KEY }}
+
+        - name: Build and push master
+          run: |
+            docker build -f script-proxy-opts/Dockerfile.proxy -t eu.gcr.io/tradeshift-base/sonar-proxy:master .
+            docker push eu.gcr.io/tradeshift-base/sonar-proxy:master
```

Comment:
Doesn't actions-docker-gcr support this? If not I guess it should?

Reply:
latest not master